custom-elements/reactions/DOMTokenList.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Custom Elements: CEReactions on DOMTokenList interface</title>
+<meta name="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+<meta name="assert" content="add, remove, toggle, replace, and the stringifier of DOMTokenList interface must have CEReactions">
+<meta name="help" content="https://dom.spec.whatwg.org/#node">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="./resources/reactions.js"></script>
+</head>
+<body>
+<div id="log"></div>
+<script>
+
+test(function () {
+    var element = defineNewCustomElement(['class']);
+    var instance = document.createElement(element.name);
+    assert_array_equals(element.log().types(), ['constructed']);
+    instance.classList.add('foo');
+    var logEntries = element.log();
+    assert_array_equals(logEntries.types(), ['attributeChanged']);
+    assert_equals(logEntries.log().name, 'class');
```

Comment:
This pattern of four assertions is repetitive; I wonder if the tests would be more readable if you add a helper which takes logEntries.log() and four parameters and have the helper do the four assertions?